csv-data-management/info.json

```diff
@@ -331,6 +331,124 @@
         }
       ],
       "open": true
+    },
+    {
+      "title": "Join and Extract Data from two CSV",
+      "operation": "join_two_csv_and_extract_data",
+      "description": "This action will extract data by 'joining' two CSV file. User can select option to deduplicate values based on column(s) provided",
+      "parameters": [
+        {
+          "title": "Type",
+          "type": "select",
+          "name": "input",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": "",
+          "options": [
+            "Attachment IRI",
+            "File IRI"
+          ]
+        },
+        {
+          "title": "First File Reference ID",
+          "type": "text",
+          "name": "file_one_value",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": ""
+        },
+        {
+          "title": "First File Column Names",
+          "type": "text",
+          "name": "file1_column_names",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": "",
+          "description": "comma separate list of columns name you want to extract from CSV File",
+          "tooltip":"comma separate list of columns name you want to extract from CSV File"
+        },
+        {
+          "title": "Number of rows to skip from First File",
+          "type": "integer",
+          "name": "numberOfRowsToSkipFirst",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": null,
+          "description": "Number of rows you want to skip from top of CSV File. Note first row will not be skipped as it has column names",
```

Comment:
Suggestion: Number of rows you want to skip from top of the CSV File. Please note, first row will not be skipped as we assume that it has column names

Reply:
Please note, first row will not be skipped as we assume that it has column names -- this is not correct, 1st row is also included. We cannot assume first row will be header. We can csv with product / reporting tool info in first few rows